tiny_dnn/core/kernels/conv2d_grad_op_avx.h

```diff
@@ -41,6 +41,52 @@ namespace kernels {
 
 // float ver
 template <typename Allocator>
+inline void accumulate_db(const index3d<serial_size_t>&        out,
+                          const std::vector<float, Allocator>& curr_delta,
+                          std::vector<float, Allocator>&       db) {
+    if (out.width_ == 1 && out.height_ == 1) {
+        size_t nblocks = out.depth_ / 8;
+        size_t remainder = out.depth_ & 7;
+        for (size_t i = 0; i < nblocks; ++i) {
+            _mm256_storeu_ps(&db[i*8], _mm256_add_ps(_mm256_loadu_ps(&db[i*8]), _mm256_loadu_ps(&curr_delta[i*8])));
```

Comment:
If you impose `spaces < 80` rule, please add it to [Wiki](https://github.com/tiny-dnn/tiny-dnn/wiki/Implementation-Guides#coding-rules) and [this document](https://github.com/tiny-dnn/tiny-dnn/blob/master/docs/developer_guides/How-to-contribute.md)

_Expect heavy changes!_